main/views/main.py

```diff
@@ -235,11 +235,17 @@ def get(self, request, *args, **kwargs):
         }
         for a in Address.objects.filter(entry=user_map):
             context["street"] = a.street
-            context["city"] = (a.city + ", " + a.state + " " + a.zipcode)
```

Comment:
p.s. these are Kyle's changes not mine - my precommit hooks just reformatted some lines here!